x-pack/test/detection_engine_api_integration/security_and_spaces/update_prebuilt_rules_package/update_prebuilt_rules_package.ts

```diff
@@ -133,6 +134,8 @@ export default ({ getService }: FtrProviderContext): void => {
       // Verify that the _perform endpoint returns the same number of installed rules as the status endpoint
       // and the _review endpoint
       const installPrebuiltRulesResponse = await installPrebuiltRules(supertest);
+      await es.indices.refresh({ index: ALL_SAVED_OBJECT_INDICES });
```

Comment:
It's quite easy to forget why `es.indices.refresh()` is needed and what part of test it's related to. It'd be nicer to have it right before an HTTP call depending on it so anybody reusing it doesn't have to keep in mind `es.indices.refresh()` has to be called before. For example `getPrebuiltRulesStatus()` should contain `es.indices.refresh()` inside.